libs/wingcompiler/src/wingc.ts

```diff
@@ -103,8 +103,14 @@ export async function load(options: WingCompilerLoadOptions) {
       .map((value) => value.trim());
 
     for (const drive of drives) {
-      // drive will be something like "C:"
-      preopens[`${drive}/`] = `${drive}\\`;
+      try {
+        let actualPath = `${drive}\\`;
+        await fs.promises.access(actualPath, fs.constants.R_OK | fs.constants.F_OK);
+        // drive will be something like "C:"
+        preopens[`${drive}/`] = actualPath;
+      } catch {
+        // can't access the drive, don't bother preopening it
+      }
```

Comment:
nit: when a function throwing starts change the control flow of the rest of the code I feel like it's sometimes clearer to refactor into a function

```suggestion
      if (await isDriveAccessible(drive)) {
        // drive will be something like "C:"
        preopens[`${drive}/`] = `${drive}\\`;
      }
```